libs/portals/admin/ids-admin/src/components/Client/AdvancedSettings.tsx

```diff
@@ -85,53 +75,81 @@ const AdvancedSettings = ({
           large
           disabled={!isSuperAdmin}
           name="requireConsent"
-          value={`${consent}`}
+          defaultChecked={inputValues.requireConsent}
+          checked={inputValues.requireConsent}
+          onChange={(e) => {
+            setInputValues({
+              ...inputValues,
+              requireConsent: e.target.checked,
+            })
+          }}
+          value={'true'}
```

Comment:
~When the value is just hardcoded string we can drop the brackets.
```suggestion
          value="true"
```